components/user/themes/UserTabs.js

```diff
@@ -9,11 +9,11 @@ import UserRepos from "../UserRepos";
 
 export default function UserTabs({ data, BASE_URL }) {
   const defaultTabs = [
-    { name: "My Links", href: "#", current: true },
-    { name: "Milestones", href: "#", current: false },
-    { name: "Testimonials", href: "#", current: false },
-    { name: "Events", href: "#", current: false },
-    { name: "Repos", href: "#", current: false },
+    { name: "My Links", href: "#", current: true, url: 'links'},
+    { name: "Milestones", href: "#", current: false, url: 'milestones' },
+    { name: "Testimonials", href: "#", current: false, url: 'testimonials' },
+    { name: "Events", href: "#", current: false, url: 'events' },
+    { name: "Repos", href: "#", current: false, url: 'repos' },
```

Comment:
```suggestion
    { name: "My Links", href: "#", current: true, url: "links" },
    { name: "Milestones", href: "#", current: false, url: "milestones" },
    { name: "Testimonials", href: "#", current: false, url: "testimonials" },
    { name: "Events", href: "#", current: false, url: "events" },
    { name: "Repos", href: "#", current: false, url: "repos" },
```
coding style does not match with project code, please install vscode prettier to fix this

Reply:
Doesn't this run in github action?

Reply:
if it is not done then someone else's commit will reflect that changes